package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "cross-env NODE_OPTIONS='--inspect' next dev",
```

Comment:
we shouldn't need this as the vscode terminal will auto-attach the debugger for any node.js process